pkgs/racket-doc/scribblings/reference/units.scrbl

```diff
@@ -547,11 +562,17 @@ information of the signatures for the unit's imports (i.e., the lexical
 information that would normally be derived from the signature reference).
 See @racket[define-signature] for more information.}
 
-@defform/subs[
-#:literals (export link)
-(define-values/invoke-unit/infer maybe-exports unit-spec)
-[(maybe-exports code:blank (export tagged-sig-spec ...))
- (unit-spec unit-id (link link-unit-id ...))]]{
+@defform[
+#:literals (export link values)
+(define-values/invoke-unit/infer unit-spec
+  maybe-exports
+  maybe-results-clause)
```

Comment:
The old order should be listed here, too, since it's still supported. I'd use `defform*`.

Reply:
I’ve done this, and I’ve added a paragraph about it in the body, so I’ve also pared down the wording in the `@history` annotation. Let me know if the updated version looks good.